Interni/NormeProgetto/components/norme-c2.tex

```diff
@@ -0,0 +1,205 @@
+\subsubsection{Aspettative}
+Durante lo svolgimento del progetto il gruppo intende mantenere un dialogo costante con il proponente, in modo da instaurare un rapporto di collaborazione e ricevere un riscontro proficuo sul lavoro svolto.  In particolare, si vuole:
+\begin{itemize}
+	\item determinare gli aspetti chiave che il prodotto dovrà soddisfare;
+	\item definire vincoli e requisiti riguardanti i processi;
+	\item stimare costi e tempistiche di lavoro;
+	\item accordarsi sulla qualifica del prodotto.
+\end{itemize}
+\subsubsection{Studio di fattibilità}
+Il \RdP{} ha il compito di organizzare riunioni tra i membri in modo da promuovere lo lo scambio di opinioni sui capitolati proposti. Una volta che viene trovato un capitolato di interesse comune, gli \anas{} provvedono alla stesura dello \SdF{} che, per ogni capitolato, indica:
+\begin{itemize}
+	\item \textbf{Informazioni generali}: vengono elencate le informazioni di base,  quali nome del progetto, il proponente e il committente;
+	\item \textbf{Descrizione e finalità del capitolato}: sintesi del prodotto, comprendente le caratteristiche principali che vengono richieste e l'obiettivo da raggiungere;
+	\item \textbf{Tecnologie interessate}: elenco delle tecnologie richieste per lo svolgimento del prodotto;
+	\item \textbf{Aspetti positivi, criticità e fattori di rischio}: esposizione delle considerazioni fatte dal gruppo riguardanti gli aspetti positivi e sulle criticità del capitolato in questione;
+	\item \textbf{Conclusioni}: esposizione delle ragioni per le quali il gruppo accetta o rifiuta il capitolato.
+\end{itemize}
+\subsubsection{Altra documentazione fornita}
+In questa sezione vengono elencati i documenti che sono stati forniti all'azienda \proponente{} e ai committenti \textit{Prof. Tullio Vardanega} e \textit{Prof. Riccardo Cardin}. La suddetta documentazione risulta essenziale in modo da tracciare le attività  svolte dal gruppo.
+I documenti forniti riguardano: 
+\begin{itemize}
+	\item \textbf{Analisi}: contenuta nell' \AdR{}, documento che esamina tutti i requisiti e i casi d'uso individuati;
+	\item \textbf{Pianificazione e consegna}: nel documento \PdP{} viene presentato un preventivo riguardante tempistiche, l' analisi dei rischi, la pianificazione delle attività e il consuntivo;
+	\item \textbf{Verifica, validazione e qualità}: all'interno del \PdQ{} vengono descritte le metodologie utilizzate per valutare e garantire la qualità del prodotto.
+\end{itemize}
+\subsubsection{Strumenti}
+\paragraph{Gantt project}
+I diagrammi di Gantt sono uno strumento molto versatile per visualizzare e tracciare tempistiche e avanzamento del lavoro.  Sono utili al \RdP{} per tenere traccia di tutte attività, tra le quali l'assegnazione di risorse,  la gestione del budget e l'analisi dei carichi di lavoro, e le dipendenze tra esse. Per la realizzazione dei suddetti diagrammi, viene utilizzato \textit{Gantt Project}.
```

Comment:
rimuovere doppio spazio prima di "Sono utili al \RdP{}"

rimuovere doppio spazio prima di "la gestione del  budget"

Cambiare
"... per tenere traccia di tutte attività ..."
in
"... per tenere traccia di tutte le attività ..."

Reply:
Fixed